airbyte-commons-worker/src/test/java/io/airbyte/workers/general/StubAirbyteMapper.java

```diff
@@ -0,0 +1,18 @@
+package io.airbyte.workers.general;
+
+import io.airbyte.protocol.models.AirbyteMessage;
+import io.airbyte.protocol.models.ConfiguredAirbyteCatalog;
+import io.airbyte.workers.internal.AirbyteMapper;
+
+public class StubAirbyteMapper implements AirbyteMapper {
```

Comment:
What effect does this class have on JSON-parsing in the orchestrator?  If the map is null, is it skipped?  We probably at least skip the record-schema validation...

Reply:
The map catalog method is minimal. 

Checking the map record method.

Reply:
record schema validation happens here https://github.com/airbytehq/airbyte/blob/master/airbyte-commons-worker/src/main/java/io/airbyte/workers/RecordSchemaValidator.java#L53 in a separate class

Reply:
without map record, we consistently hit 22 - 27 MB/s.
with the map record, we consistently hit 18 - 25 MB/s.

I think there is something here.